src/test/scala/uk/gov/nationalarchives/consignmentexport/MainSpec.scala

```diff
@@ -120,9 +124,32 @@ class MainSpec extends ExternalServiceSpec {
     ex.getMessage should equal(s"Incomplete details for user $keycloakUserId")
   }
 
+  "the export job" should "should publish the step function success token if task token argument provided" in {
+    setUpValidExternalServices()
+
+    val consignmentId = UUID.fromString("50df01e6-2e5e-4269-97e7-531a755b417d")
+    val taskTokenValue = "taskToken1234"
+    putFile(s"$consignmentId/7b19b272-d4d1-4d77-bf25-511dc6489d12")
+    Main.run(List("export", "--consignmentId", consignmentId.toString, "--taskToken", taskTokenValue)).unsafeRunSync()
+
+    wiremockSfnServer.getAllServeEvents.size() should be(1)
+    val eventRequestBody = wiremockSfnServer.getAllServeEvents.get(0).getRequest.getBodyAsString
+    eventRequestBody.contains(taskTokenValue) should be(true)
+
+    //check rest of process was completed successfully
+    val objects = outputBucketObjects().map(_.key())
+    objects.size should equal(2)
```

Comment:
Minor thing again but if you're only checking the size of the list, you don't need the `.map(_.key())` bit here.

Reply:
I blame "copy and paste" ;)